### PR TITLE
Pull request for sileht/subscription

### PR DESCRIPTION
**Pull request by sileht**
Don't return private repo to UI
Separate redis for rq and cache
Enable private repo for subscribed install

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now abeb540ed9da7ed58ee1eeb5bef342b85e3c5651

File: mergify_engine/engine.py [hunk 1]
```diff
@@ -298,8 +276,7 @@ def proceed_queue(self, branch, **extra):
             if p.mergify_engine["combined_status"] == "success":
                 # rebase it and wait the next pull_request event
                 # (synchronize)
-                updater_token = self.get_updater_token()
-                if not updater_token:
+                if not self._subcription["token"]:
```
Comment (jd) [inline mergify_engine/engine.py]: subscription, typo

File: mergify_engine/utils.py [hunk 2]
```diff
@@ -112,6 +124,32 @@ def get_installation_id(integration, owner):
             return install["id"]
 
 
+def get_subscription(r, installation_id):
+    sub = r.hgetall("subscription-cache-%d" % installation_id)
+    if sub is None:
+        LOG.info("Subscription for %s not cached, retrieving it..." %
+                 installation_id)
+        resp = requests.get("https://mergify.io/engine/token/%s" %
```
Comment (jd) [inline mergify_engine/utils.py]: URL seems wrong, should be `/engine/installation/%d`

File: mergify_engine/engine.py (hunk 1 above)
Comment (sileht) [reply mergify_engine/engine.py]: done

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 7fd271e80a2a22425c186ec16ef01202cb248a96

File: mergify_engine/utils.py (hunk 2 above)
Comment (sileht) [reply mergify_engine/utils.py]: Thx, I didn't notice it change

**Pull request merged**